.gitignore

```diff
@@ -63,3 +63,5 @@ bag/
 .\#*
 \#*\#
 
+# Workspace files
+home/arcturus/ros2_ws/
```

Comment:
Probably don't need to add this to gitignore. It's not really expected that the user creates a custom ros2_ws and you can also just not push the workspace.